### PR TITLE
#no number java doc thorben

### PR DESCRIPTION
**Pull request by ThorbenKuck**

### CODE REVIEW

File: src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java [hunk 1]
```diff
@@ -25,13 +31,13 @@ public String getPrefix() {
...
 	 */
-	String getCaller() {
+	private String getCaller() {
 		final StackTraceElement[] stackTraceElements = Thread.currentThread().getStackTrace();
 		for (StackTraceElement stackTraceElement : stackTraceElements) {
-			if (! stackTraceElement.getClassName().equals(CallerReflectionLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(NetComLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(CallerTraceSystemDefaultStyleLogging.class.getName())
-					&& ! stackTraceElement.getClassName().equals(SystemDefaultStyleLogging.class.getName())
+			if (!stackTraceElement.getClassName().equals(CallerReflectionLogging.class.getName())
```
Comment (Minification) [inline src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java]: Ugh

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,7 +71,9 @@ If you want to provide a new feature or a bug-fix, please follow these steps:
 3) Fix the bug/develop the new feature based on your design.
 4) Write a test. Test any case you could come up with.
 5) Open a pull-request based on the provided pull-request template onto the branch your branch branched from.
-6) If your pull-request get's denied, start over from 3.
+6) Add an direct contributor as an reviewer to your pull-request
```
Comment (Minification) [inline CONTRIBUTING.md]: *a direct contributor 
*a reviewer

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,7 +71,9 @@ If you want to provide a new feature or a bug-fix, please follow these steps:
...
+7) Add an direct contributor as an assignee to your pull-request
```
Comment (Minification) [inline CONTRIBUTING.md]: *a direct contributor

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -53,7 +71,9 @@ If you want to provide a new feature or a bug-fix, please follow these steps:
...
+8) If your pull-request get's denied, start over from 3.
```
Comment (Minification) [inline CONTRIBUTING.md]: *gets

File: src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java (hunk 1 above)
Comment (ThorbenKuck) [reply src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java]: Yep, this is analysing the stacktrace and it is by no mean beautifull

Comment (ThorbenKuck) [reply src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java]: That, by the way, is the reason, why it should be used with great care.

Comment (Minification) [reply src/main/java/com/github/thorbenkuck/netcom2/logging/CallerTraceSystemDefaultStyleLogging.java]: Really no way to make this better?

**Pull request merged**